src/bindings/js/node/tests/unit/core.test.js

```diff
@@ -5,6 +5,7 @@
 const { addon: ov } = require('../..');
 const assert = require('assert');
 const { describe, it } = require('node:test');
+const { getModelPath } = require('./utils.js');
 
 describe('ov.Core tests', () => {
```

Comment:
```suggestion
const { describe, it, before } = require('node:test');
const { testModels, isModelAvailable, getModelPath } = require('./utils.js');

describe('ov.Core tests', () => {

  before(async () => {
    await isModelAvailable(testModels.testModelFP32);
  });
```
This change is necessary after you merge changes from master branch ;)

Reply:
Done